packages/synapse-react-client/src/components/MuiContainer.tsx

```diff
@@ -0,0 +1,31 @@
+import { StyledComponent } from '@emotion/styled'
+import { Paper } from '@mui/material'
+import { PaperProps, styled } from '@mui/material'
+
+/**
+ * Drop in MUI substitute class for Bootstrap Container component
+ */
+export const MuiContainer: StyledComponent<PaperProps> = styled(Paper, {
```

Comment:
New StyledComponent (based on Paper) that sets the width dynamically based on the current theme breakpoint

Reply:
May want to base this on `Box` instead of `Paper`, and just wrap in `Paper` (with `px: 15px`) when appropriate (with `px: 15px`)

Reply:
You get slightly different behavior if you use [MUI's built-in](https://mui.com/material-ui/api/container/) `Container` with the `fixed` prop, but would that be 'good enough' as a drop-in replacement?